src/plugins/Workers/TransactionSyncStreamWorker/TransactionSyncStreamWorker.spec.js

```diff
@@ -0,0 +1,60 @@
+const { expect } = require('chai');
+const TransactionSyncStreamWorker = require('./TransactionSyncStreamWorker');
+const Storage = require('../../../types/Storage/Storage');
+const EventEmitter = require('events');
+
+describe('Workers - TransactionSyncStreamWorker', function suite() {
```

Comment:
It seems there is no point to implement unit test here. Let's move it to `tests/integration` and cover all cases using `FixtureTransport`